Techome/Techome/Views/AddCaffeine/SearchCaffeineView.swift

```diff
@@ -48,52 +50,64 @@ struct SearchCaffeineView: View {
 }
 
 struct RecentlyAddedCaffeine: View {
+    
     var body: some View {
         
-        
         VStack {
             Text("최근 추가한 카페인")
                 .font(.body)
-                .foregroundColor(.black)
+                .foregroundColor(.customBlack)
                 .fontWeight(.semibold)
                 .frame(maxWidth:.infinity, alignment: .leading)
                 .padding(.horizontal, SearchCaffeineViewLayoutValue.Padding.cardTitleHorizontalPadding)
             
             VStack(spacing: .zero) {
-                ForEach (0 ..< 7) { _ in
-                    CaffeineRecordRow()
+                if IntakeManager.shared.getRecentRecords(count: 6).count != 0 {
+                    
+                    VStack(spacing: .zero) {
+                        ForEach (IntakeManager.shared.getRecentRecords(count: 6), id: \.self) { record in
+                            CaffeineRecordRow(recentRecord: record)
+                        }
+                    }
+                    .background(RoundedRectangle(cornerRadius: SearchCaffeineViewLayoutValue.Radius.card)
+                        .foregroundColor(.white)
+                        .shadow(color: .secondaryShadowGray, radius: SearchCaffeineViewLayoutValue.Radius.shadow, x: .zero, y: .zero)
+                        )
+                } else {
+                    Text("최근 추가된 카페인이 없습니다.")
+                        .padding(.top, 150)
+                        .foregroundColor(.customBlack)
                 }
             }
-            .background(RoundedRectangle(cornerRadius: SearchCaffeineViewLayoutValue.Radius.cardRadius)
-                .foregroundColor(.white)
-                .shadow(color: .secondaryTextGray, radius: SearchCaffeineViewLayoutValue.Radius.shadowRadius, x: .zero, y: .zero))
-            
         }
         .padding(EdgeInsets(top: SearchCaffeineViewLayoutValue.Padding.cardVerticalPadding, leading: .zero, bottom: .zero, trailing: .zero))
     }
 }
 
 struct CaffeineRecordRow: View {
+    
+    var recentRecord: IntakeRecord
```

Comment:
```suggestion
    private var recentRecord: IntakeRecord
```

접근 제한자 설정이 필요해보여요 ~~

Reply:
접근 제한자 설정이 필요하군요!! 설정하겠습니당

Reply:
아! 상위뷰에서 값을 전달해야하기 때문에 접근 제한자 설정을 하지 않았습니다 ~~